include/modem/location.h

```diff
@@ -161,14 +161,56 @@ struct location_datetime {
 struct location_data_details_gnss {
 	/** Number of satellites tracked at the time of event. */
 	uint8_t satellites_tracked;
+	/** Number of satellites used for a fix. */
+	uint8_t satellites_used;
 	/** PVT data. */
 	struct nrf_modem_gnss_pvt_data_frame pvt_data;
+	/**
+	 * GNSS search time in milliseconds.
+	 *
+	 * This is the time since last start of GNSS operation to the fix or timeout
+	 * including any time LTE has blocked GNSS.
+	 *
+	 * @ref pvt_data has nrf_modem_gnss_pvt_data_frame.execution_time which indicates
+	 * cumulative GNSS execution time since last start excluding any time LTE has blocked
+	 * GNSS.
+	 */
+	uint32_t search_time;
```

Comment:
@tokangas, @hiltunent, how should we name this? TTF is not good because we use it also for search time when timeout occurs.

Reply:
"search" is not particularly good but I had to put something 😄
Somewhere "used_time" was used which I feel is worse.
"fix_time" or "ttf" cannot be used as this is for timeout cases too.
I hope @hiltunent will help us 😄

Reply:
I'd vote for elapsed_method_time and elapsed_gnss_time

Reply:
Changed `gnss_search_time` to `elapsed_time_gnss` and `method_search_time` to `elapsed_time_method`.